src/realm/util/backtrace.cpp

```diff
@@ -0,0 +1,65 @@
+#include <realm/util/backtrace.hpp>
```

Comment:
Missing copyright header.

Reply:
Oh you put those in cpp files too?

Reply:
Hrm not sure actually, I think we do it inconsistently 🙄